TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -122,8 +121,8 @@ public override bool Visit(Move move)
                         true; //Do not continue, the variables hasn't been found. An error will be raised later by CheckVariable()
                 }
 
-                var fromVariableChildren = fromVariable.Children.Where(c => c?.Name != null);
-                var toVariableChildren = toVariable.Children.Where(c => c?.Name != null);
+                var fromVariableChildren = fromVariable.Children.Where(c => c?.Name != null).AsQueryable();
```

Comment:
Why use AsQueryable here?

Reply:
AsQueryable() -  query that returns a sequence of values do not consume the target data until the query object is enumerated - also known as deferred execution.

For more references:
https://msdn.microsoft.com/en-us/library/bb507003(v=vs.110).aspx
https://msdn.microsoft.com/en-us/library/system.linq.enumerable(v=vs.110).aspx